docs/experiments-nfcorpus.md

```diff
@@ -71,6 +71,8 @@ This command-line invocation does the trick:
 tail -n +2 collections/nfcorpus/qrels/test.tsv | sed 's/\t/\tQ0\t/' > collections/nfcorpus/qrels/test.qrels
 ```
 
+Q0 instead of 0 (historical artifact mentioned in [Anserini: Start Here](https://github.com/castorini/anserini/blob/master/docs/start-here.md)) marks as trec format in pyserini to avoid extraneous conversion from msmarco in [trec_eval.py](https://github.com/castorini/pyserini/blob/master/pyserini/eval/trec_eval.py).
```

Comment:
Thanks for noticing. Actually, this isn't technically correct... so let's just remove?

Reply:
Oh right, I read the code over and that Q0 is for checking the format of the search output. I'll remove the lines. Then, adding in the qrels the Q0 or 0 does not really matter here?